### PR TITLE
Update README & add CONTRIBUTING

### PR DESCRIPTION
**Pull request by antonyoneill**
# What
Rejig of the README & add separate CONTRIBUTING

# Why
The screenshots were out of date, and a CONTRIBUTING file should be useful for people onboarding onto the project

# Screenshots

# Notes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+For an in-depth set up guide, see the [Development Guide](docs/development/README.md).
+
+## Clean Architecture
+
+We try to follow Clean Architecture patterns to ensure our code is clean and concise. Clean Architecture defines clear boundaries between different parts of the service, from external `Gateways` to `usecase`s that build on business rules to perform tasks.
+
+## Testing
+
+We believe in Test Driven Development. This ensures we keep a high code coverage metric, and forces us to understand the crux of the problem that we're trying to solve. TDD may slow us down, but in the long run we end up with a more stable, maintainable solution.
```
Comment (chubberlisk) [inline CONTRIBUTING.md:19]:
```suggestion
We believe in Test Driven Development (TDD). This ensures we keep a high code coverage metric, and forces us to understand the crux of the problem that we're trying to solve. TDD may slow us down, but in the long run we end up with a more stable, maintainable solution.
```

File: README.md [hunk 2]
```diff
@@ -2,38 +2,37 @@
 
 This service allows ward staff to schedule a visit for a patient. Allowing face to face visits for visitors who are unable to visit in person.
 
+It includes an administration panel which provides self-service functionality for adding new Trusts, Hospitals, and Wards.
```
Comment (chubberlisk) [inline README.md:5]: Should we also update our glossary with these terms and pages? 🤔

File: README.md [hunk 3]
```diff
@@ -2,38 +2,37 @@
...
-| ------------------------------------------------------------------------ | ------------------------------------------------------------------- |
-| ![Ward Staff see list of visits sorted by Today, Upcoming, and Past 12 hours](docs/images/WardStaff_Overview.png) | ![Ward Staff start a visits](docs/images/WardStaff_StartScreen.png) |
-
-| Visitor receives SMS notifications                                 | Visitor joins a visit                                         | Visitor ends a visit                                       |
-| ------------------------------------------------------------------ | ------------------------------------------------------------- | ---------------------------------------------------------- |
-| ![Visitor receives SMS notifications](docs/images/Visitor_SMS.png) | ![Visitor joins a visit](docs/images/Visitor_StartScreen.png) | ![Visitor ends a visit](docs/images/Visitor_EndScreen.png) |
+| ![Trust Admin can add new hospitals and wards](docs/images/TrustAdmin_01_AddNew.png)            | ![Trust Admin can edit and delete hospitals and wards](docs/images/TrustAdmin_02_ManageExisting.png) |
```
Comment (chubberlisk) [inline README.md:32]: 😍 

How does one update these screenshots? Looks like it's taken from a slide deck.

File: README.md [hunk 4]
```diff
@@ -46,169 +45,13 @@ The following video chat capabilities are currently supported:
...
-```
-
-You can do a dry-run to view the changes that will be applied without making any changes (for both up and down migrations)
-
-```bash
-npm run dbmigratedry up
-```
-
-To update the `db/schema.sql` file:
+## Development
```
Comment (chubberlisk) [inline README.md:52]: I think Development should go before Contributing. My thoughts are you'd need to set up the repo and get the app running before thinking about adding to the codebase.

File: README.md [hunk 5]
```diff
@@ -46,169 +45,13 @@ The following video chat capabilities are currently supported:
...
 
-```bash
-pg_dump -d nhs-virtual-visit-dev -s > db/schema.sql
-```
+Please read our separate [Development Guide](./docs/development/README.md).
```
Comment (chubberlisk) [inline README.md:54]: My preference would be to keep this in the README and extract out less likely to be used sections like "Scheduling regular database cleanup" and "Building a production version" into separate files so it's not too long. I think it's just what I'm used to seeing in README, happy to go with this though.

File: README.md (hunk 2 above)
Comment (antonyoneill) [reply README.md:5]: Yap!

File: README.md (hunk 3 above)
Comment (antonyoneill) [reply README.md:32]: It is.. https://docs.google.com/presentation/d/1KaHYSZzcdFJ1oOCZdiPfZCXv9uAEOeE8EvkIjD-mId8/edit

File: README.md (hunk 4 above)
Comment (antonyoneill) [reply README.md:52]: True!

File: README.md (hunk 5 above)
Comment (antonyoneill) [reply README.md:54]: I think we're starting to share this repo with potential trusts, so I'm keen to make the README more of an overview than a detailed page for developers

File: README.md (hunk 2 above)
Comment (antonyoneill) [reply README.md:5]: I've removed the pages, as I'm not convinced there's enough value documenting those anymore (we initially added it to keep the routes consistent)

File: README.md (hunk 3 above)
Comment (antonyoneill) [reply README.md:32]: Added a link to the README

**Pull request merged**